Documentation/FloorFilter/README.md

```diff
@@ -4,7 +4,9 @@ The `FloorFilter` component simplifies visualization of GIS data for a specific
 
 The ArcGIS Runtime SDK currently supports filtering a 2D floor aware map based on the sites, buildings, or levels in the map.
 
-###Behavior:
```

Comment:
I noticed the "Usage" section header below has no colon. Should the section headers match - either colon or no colon?

Reply:
Good catch.  I made the change (to no `:`) here and in a couple of other spots.